### PR TITLE
Jason faq

### PR DESCRIPTION
**Pull request by simonnguyen3054**
I commented out About Us because we dont need it for now.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -22,9 +22,14 @@ import DealItem from "./containers/Deals/DealItem";
...
 import TrackingNumber from "./containers/Deals/TrackingNumber";
+import Faq from "./containers/FooterPages/Faq/Faq"
+import TermsOfServ from "./containers/FooterPages/TermsOfServ/Terms"
+import Returns from "./containers/FooterPages/Returns/Returns"
+import CookiesPolicy from "./containers/FooterPages/CookiesPolicy/CookiesPolicy"
+
 import Chat from "./containers/Chat";
 
-// import Layout from "./components/Layout";
+import Layout from "./containers/Layout";
```
Comment (avakrishn) [inline client/src/App.js]: @simonnguyen3054 Why was this changed from components to containers? Was the styling as well as the component moved from the the component folder to the containers folder?

File: client/src/App.js [hunk 2]
```diff
@@ -67,6 +72,13 @@ class App extends Component {
 
               <Route exact path='/chat' component={Chat}/>
 
+              {/* <Route exact path='/AboutUs' component={AboutUs}/> */}
+              <Route exact path='/TermsOfServ' component={TermsOfServ}/>
```
Comment (avakrishn) [inline client/src/App.js]: @simonnguyen3054 Can we change the path name to '/TermsOfService'

File: client/src/components/Layout/Footer/Footer.js [hunk 3]
```diff
@@ -1,12 +1,60 @@
...
+            <Link className="Footer-link" to="/Returns">
+              AcceptMyReturns
+            </Link>
+          </span>
+        </div>
 
+        <div className="Footer-terms">
+          <h4 className="Footer-linkTitle">Terms</h4>
+          <span>
+            <Link className="Footer-link" to="/TermsOfServ">
```
Comment (avakrishn) [inline client/src/components/Layout/Footer/Footer.js]:
```suggestion
            <Link className="Footer-link" to="/TermsOfService">
```

File: client/src/App.js (hunk 2 above)
Comment (avakrishn) [inline client/src/App.js]:
```suggestion
              <Route exact path='/TermsOfService' component={TermsOfServ}/>
```

Comment (avakrishn) [reply client/src/App.js]: @simonnguyen3054 With Github's new commit suggestions if you accept the suggestion, you just need to click on the button `Commit suggestion`  above and then it will automatically change the commit. That makes it easier and the person who suggests the change also gets credit for the suggestion.

File: client/src/App.js (hunk 1 above)
Comment (simonnguyen3054) [reply client/src/App.js]: hmm It should be components still. I think I changed it accidently
but it should be commented out.
